internal/binder/function/funcs_agg.go

```diff
@@ -260,6 +290,11 @@ func registerAggFunc() {
 	builtins["var"] = builtinFunc{
 		fType: ast.FuncTypeAgg,
 		exec: func(ctx api.FunctionContext, args []interface{}) (interface{}, bool) {
+			for _, arg := range args {
```

Comment:
Looks like this is the same for all func, can we add this to callValuer: check if any arg is nil, if find any, just return nil

Reply:
updated.